### PR TITLE
batch.go minor editorial tweaks

### PR DESCRIPTION
**Pull request by barriebyron**
## Description

Going through the comments one file at a time so that we can close the larger set of open docs and editorial issues and PRs. 
Closes part of #177

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: x/liquidity/keeper/batch.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func (k Keeper) DeleteAndInitPoolBatch(ctx sdk.Context) {
 			height := ctx.BlockHeight()
 
 			// reinitialize remaining batch msgs
-			// In the case of BatchSwapMsgs, it is often fractional matched or has not yet expired since it has not passed ExpiryHeight.
+			// In the case of BatchSwapMsgs, it is often fractionally matched or has not yet expired since it has not passed ExpiryHeight.
```
Comment (barriebyron) [inline x/liquidity/keeper/batch.go:38]: ❓ what do we mean by "it"
do we mean  "the batch messages are fractionally matched"? 
or is it a message in the batch that is fractionally matched? I'm not clear

Comment (kogisin) [reply x/liquidity/keeper/batch.go:38]: > ❓ what do we mean by "it"

It refers to the user's swap order.

**Review by kogisin — APPROVED** [review]
There seems to be some conflicts. I'll fix it.

**Comment by kogisin** [comment]
The `develop` branch that @barriebyron use on this PR looks very old. It has 135 commits behind. Merging with the latest `develop` branch doesn't seem to catch all the updates. Since this edit is trivial, it might be better to have another PR that is based of the latest `develop` branch. I'll make another PR for this.

**Comment by kogisin** [comment]
The `develop` branch that @barriebyron use on this PR looks very old. It has 135 commits behind. Merging with the latest `develop` branch doesn't seem to catch all the updates. Since this edit is trivial, it might be better to have another PR that is based of the latest `develop` branch. I'll make another PR for this.

**Comment by barriebyron** [comment]
I need remedial git lessons @kogisin !! thanks for fixing me up here

**Pull request closed without merging**